lib/methods/platform/getDataContract/GetDataContractResponse.js

```diff
@@ -24,11 +24,11 @@ class GetDataContractResponse extends AbstractResponse {
    * @return {GetDataContractResponse}
    */
   static createFromProto(proto) {
-    const metadata = proto.getMetadata() ? new Metadata(proto.getMetadata().toObject()) : null;
+    const metadata = proto.getMetadata() ? new Metadata(proto.getMetadata().toObject()) : undefined;
 
     return new GetDataContractResponse(
-      metadata,
       proto.getDataContract() ? Buffer.from(proto.getDataContract()) : undefined,
```

Comment:
I guess the data contract always must be there. What is the point of this condition?

Reply:
I suppose we don't want to have fatal error if getDataContract returns undefined

Reply:
We can throw another error if it's empty.